templates/todo/common/tests/todo.spec.ts

```diff
@@ -21,6 +21,13 @@ test("Create and delete item test", async ({ page }) => {
   await expect(page.locator(`text=${guid}`).first()).toBeVisible()
 
   await page.locator(`text=${guid}`).click();
+
+  const itemMoreOperation = await page.$('button[role="menuitem"]:has-text("")');
```

Comment:
Consider using the `\uXXXX` syntax instead of the actual codepoint in the string?

Reply:
Agreed .